TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs

```diff
@@ -58,10 +59,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<User>().Property(ap => ap.User_ID).UseIdentityColumn(1, 1).IsRequired().HasColumnName("user_id");
             //COLUMN SETTINGS
             modelBuilder.Entity<User>().Property(ap => ap.Username).IsRequired(true).IsUnicode().HasMaxLength(100).HasColumnName("username");
-            modelBuilder.Entity<User>().Property(ap => ap.User_Password).IsRequired(true).HasMaxLength(100).HasColumnName("user_password");
+            modelBuilder.Entity<User>().Property(ap => ap.User_Token).IsRequired(true).HasMaxLength(100).HasColumnName("user_token");
```

Comment:
The database won't save password anymore. Instead, it saves user-token released from Auth0 Server.